pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -42,7 +42,7 @@ import (
 
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/cloudprovider"
-	"k8s.io/kubernetes/pkg/credentialprovider/aws"
+	aws_credentials "k8s.io/kubernetes/pkg/credentialprovider/aws"
```

Comment:
This is a confusing alias (I would have guessed "github.com/aws/aws-sdk-go/aws/credentials", but I can't think of a better one that isn't 30 characters long).